README.md

```diff
@@ -115,8 +115,9 @@ Supported Languages
 -------------------
 
 - English
-- Spanish
```

Comment:
Why is Spanish removed? Isn't that: https://github.com/lazybird/django-solo/tree/master/solo/locale/es/LC_MESSAGES ?

Reply:
Not remove it. Just sort it to make it looks good.